test/update_dev_build.sql

```diff
@@ -5,75 +5,54 @@ VALUES (2, 1, 40111002 ,0, "SUPPLIER'S ACCOUNT 1"),
         (2, 1, 40111003 ,0, "SUPPLIER'S ACCOUNT 2");
 
 
+
 -- creditor group
+SELECT @account_idSup1 :=  id
```

Comment:
Your approach is so technical, it can be useful to just add the `id` of account in the insertion query. 

```sql
SET @account_idSup1 = 300;
SET @account_idSup2 = 301;
INSERT INTO account (id, type_id, enterprise_id, number, parent, label) VALUES 
(@account_idSup1, 2, 1, 40111002 ,0, "SUPPLIER'S ACCOUNT 1"),
(@account_idSup2, 2, 1, 40111003 ,0, "SUPPLIER'S ACCOUNT 2");
```

In this case code is simple and clear.